.github/actions/verify-submodule-pull-request/action.yml

```diff
@@ -25,10 +25,16 @@ runs:
         New-Item -ItemType File -Force $Profile | Out-Null
         Get-Content -Raw ${{ github.action_path }}/functions.ps1 >> $Profile
 
+    # We are using env for the title input in these steps because it realistically could contain quotes apostrophes or
```

Comment:
```suggestion
    # We are using env for the title input in these steps because it realistically could contain quotes, apostrophes, or
```

Reply:
[Serial comma](https://www.grammarly.com/blog/what-is-the-oxford-comma-and-why-do-people-care-so-much-about-it/) or not? I'm in favor.

Reply:
I don't think it's applicable here, because "or even both" doesn't create that kind of ambiguity. Also it confuses any comma counting tooling (we have that for markdown for example) that expect common comma usage.